index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Sabrina's Blog</title>
+  <link rel="stylesheet" type="text/css" href="css/normalize.css">
+  <link rel="stylesheet" type="text/css" href="css/skeleton.css">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+
+<body>
+  <header>
+    <h1>Sabrina Tee</h1>
+    <nav id="navbar">
+      <ul>
+        <li class="activePage"><a href="#"><span>HOME</span></a></li>
+        <li><a href="#"><span>ABOUT</span></a></li>
+        <li><a href="#"><span>PORTFOLIO</span></a></li>
+        <li><a href="#"><span>RESUME</span></a></li>
+        <li><a href="#"><span>CONTACT ME</span></a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <article id="blogPosts">
```

Comment:
I'd suggest wrapping the articles in a <section> tag and then attaching the id="blogPosts" to that. What this is doing now is creating a bunch of article elements with id="blogPosts"

![screen shot 2015-12-01 at 2 56 54 pm](https://cloud.githubusercontent.com/assets/3937302/11517098/521eaf92-983c-11e5-98f9-cac9ae089e9d.png)

You should only attach an id to one specific element.